js/classes/vector.js

```diff
@@ -99,6 +115,36 @@ class Vector {
         return Vector.unit(Vector.subtract(b, a));
     };
 
+    /**
+     * Reflects a Vector across another Vector.
+     * 
+     * @param {Vector} a the Vector to reflect
+     * @param {Vector} b the Vector across which to reflect a
+     * @returns {Vector} the reflection of a across b
+     */
+    static reflect(a, b) {
+        return Vector.subtract(a, Vector.multiply(b, 2 * Vector.dot(a, b)));
+    }
+
+    static get HORIZONTAL() {
```

Comment:
Maybe put these constants above all of the methods?

Reply:
Sounds good